build_tools/python/e2e_test_framework/unique_ids.py

```diff
@@ -21,6 +21,7 @@
 MODEL_PERSON_DETECT_INT8 = "bc1338be-e3df-44fd-82e4-40ba9560a073"
 MODEL_EFFICIENTNET_INT8 = "4a6f545e-1b4e-41a5-9236-792aa578184b"
 MODEL_MINILM_L12_H384_UNCASED_INT32_SEQLEN128 = "ecf5c970-ee97-49f0-a4ed-df1f34e9d493"
+MODEL_RESNET50_TF_FP32 = "c393b4fa-beb4-45d5-982a-c6328aa05d08"
```

Comment:
How was the unique id generated?

Reply:
Each ID should be generated from uuid.uuid4().

Reply:
This is actually in the docstring above, but GitHub really aggressively collapses things